release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -334,6 +334,20 @@ For more information on configuration drift, see xref:../post_installation_confi
 
 You can now enable link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html[Linux control groups version 2] (cgroups v2) on specific nodes in your cluster. The {product-title} process for enabling cgroups v2 disables all cgroups version 1 controllers and hierarchies. The {product-title} cgroups version 2 feature is in Developer Preview and is not supported by Red Hat at this time.
 
+[id="ocp-4-10-node-health-check-operator"]
+==== Node Health Check Operator enhancements
+
+The Node Health Check Operator provides these new enhancements:
+
+* Provides support for disconnected mode
+* Avoids conflicts with machine health check. For more information, see xref:../nodes/nodes/eco-node-health-check-operator.adoc#preventing-nhc-conflict-with-mhc_node-health-check-operator[Preventing node health check conflicts with machine health check]
+
+[NOTE]
+====
+To upgrade to the latest version of the Node Health Check Operator, you must manually change the channel name from `alpha` to `candidate`.
```

Comment:
Will folks know where/how to set the channel name?

Reply:
Good question.
Maybe change to `... update the channel name in the Subscription...`? Should at least give a hint where that channel name is

Reply:
@mburke5678 and @slintes - I understand. I think based on your feedback, it might be appropriate to update it in the main docs where we mention [channel name in the Subscription YAML](https://github.com/openshift/openshift-docs/pull/41724/files#diff-88f044abbb2615d4b62f74f1465525477e6571d9e193b910016a4b3892df2499R66-R73). Hope that's ok.

Reply:
Do we need a small module on how to update?

Reply:
@mburke5678 I am not sure if we would need a new module for changing a channel name since it is covered here. Maybe I could add these as additional resources. WDYT?

- [Changing the update channel for an Operator](https://docs.openshift.com/container-platform/4.9/updating/understanding-upgrade-channels-release.html#switching-between-channels_understanding-upgrade-channels-releases) 
- [Switching between channels](https://docs.openshift.com/container-platform/4.9/updating/understanding-upgrade-channels-release.html#switching-between-channels_understanding-upgrade-channels-releases)